EMS/helpers/src/Html/MimeTypes.php

```diff
@@ -0,0 +1,10 @@
+<?php
+
+declare(strict_types=1);
+
+namespace EMS\Helpers\Html;
+
+class MimeTypes
```

Comment:
This looks like an enum: 

Example: https://github.com/ems-project/elasticms/blob/09319673e1ea7a826e40355da57adaf003508c16/EMS/core-bundle/src/Core/Revision/Task/TaskStatus.php#L7